lib/public/views/About/Overview/index.js

```diff
@@ -11,28 +11,110 @@
+        name: 'Status',
+        format: (statuses) => h(
+            'div.flex-row.flex-wrap.justify-start',
+            Object.entries(statuses).map(([name, value]) =>
+                value ? statusBadgeSuccess(name) : statusBadgeFail(name)),
+        ),
+    },
+};
+
+/**
+ * The title banner for the service details displaying the name of the service, its
+ * version, and a badge for each status.
+ * @param {string} serviceName the name of the service
+ * @param {string} version the version of the service e.g. 1.2.3
+ * @param {Object} status an object {statusName: boolean}
+ * @returns {vnode} a div with the title, version, and status of the service
+ */
+const serviceDetailsPanelTitle = (serviceName, version) =>
+    h('div.flex-row.flex-wrap.justify-between.bg-gray-light.p2.g2', [
+        h('div.flex-row.g2.items-center ', [
+            h('h4', serviceName),
+            version
+                ? detailsFields.version.format(version)
+                : '',
+        ]),
+    ]);
+
+/**
+ * Displays panels containing information about each service.
+ * @param {RemoteData} appInfo a RemoteData containing information about the bookkeeping app
+ * @param {RemoteData} dbInfo a RemoteData containing information about the database
+ * @returns {vnode} a grid of panels, one for each service
+ */
+const servicesInfo = (appInfo, dbInfo) => {
+    const services = [
+        { name: 'Bookkeeping', infoRemoteData: appInfo },
+        { name: 'Database', infoRemoteData: dbInfo },
     ];
 
-    const aboutColumns = {
-        type: { name: 'Type', size: 'cell-s', visible: true },
-        version: { name: 'Version', size: 'cell-s', visible: true },
-        hostname: { name: 'Hostname', size: 'cell-s', visible: true },
-        port: { name: 'Port', size: 'cell-s', visible: true },
-    };
+    return services.map((s) => h(`div#${s.name}.flex-column.shadow-level1.flex-grow`, [
+        serviceDetailsPanelTitle(
+            s.name,
+            s.infoRemoteData.isSuccess() ? s.infoRemoteData.payload.version : '',
+            s.infoRemoteData.isSuccess() || s.infoRemoteData.isFailure() ? s.infoRemoteData.payload.status : '',
+        ),
+        h('div.p2', s.infoRemoteData.match({
+            NotAsked: () => 'Not Asked',
+            Loading: () => 'Fetching Details...',
+            Success: (payload) => h('div', detailsFields.status.format(payload.status)),
+            Failure: (payload) => h('.flex-column.g2', [
+                h('', detailsFields.status.format(payload.status)),
+                h('.danger', 'Error fetching details, please contact an administrator.'),
+            ]),
+        })),
+    ]));
+};
 
-    return h('.flex-row.pv2.w-100', table(data, aboutColumns));
+/**
+ * Returns a view of information panels for each service
+ * @param {object} model Pass the model to access the defined functions
+ * @returns {vnode} the view of services
+ */
+const aboutOverview = (model) => {
+    const { appInfo } = model.aboutModel;
```

Comment:
You can extract both `appInfo` and `dbInfo` in the same statement.
`const {appInfo, dbInfo} = model.aboutModel`
or 
`const {aboutModel: {appInfo, dbInfo}} = model`